src/device-registry/models/Device.js

```
@@ -714,6 +714,274 @@
+  if (timeFrame) {
+    pipeline.push({
+      $group: {
+        _id: {
+          network: "$network",
+          timeFrame: {
+            $switch: {
+              branches: [
+                {
+                  case: { $eq: [timeFrame, "daily"] },
+                  then: {
+                    $dateToString: { format: "%Y-%m-%d", date: "$$NOW" },
```

Biome

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)